tests/unit/lms/views/predicates/_lti_launch_test.py

```diff
@@ -16,55 +16,38 @@
 class TestDBConfigured:
     @pytest.mark.parametrize("value,expected", [(True, True), (False, False)])
     def test_when_theres_a_matching_assignment_config_in_the_db(
-        self, pyramid_request, value, expected, ModuleItemConfiguration
+        self, pyramid_request, value, expected
     ):
-        ModuleItemConfiguration.get_document_url.return_value = "test_document_url"
+        mic = factories.ModuleItemConfiguration()
         pyramid_request.params = {
-            "resource_link_id": "test_resource_link_id",
-            "tool_consumer_instance_guid": "test_tool_consumer_instance_guid",
+            "resource_link_id": mic.resource_link_id,
+            "tool_consumer_instance_guid": mic.tool_consumer_instance_guid,
         }
         predicate = DBConfigured(value, mock.sentinel.config)
 
         result = predicate(mock.sentinel.context, pyramid_request)
 
-        ModuleItemConfiguration.get_document_url.assert_called_once_with(
-            pyramid_request.db,
-            "test_tool_consumer_instance_guid",
-            "test_resource_link_id",
-        )
```

Comment:
I _massively_ prefer this way of going about things. We aren't specifying _how_ it get this done any more, just that it does. If this class is refactored to do this in a different way this test won't fail.